### PR TITLE
Change Firebase repository

### PR DESCRIPTION
**Pull request by wmleler**
Also, a few changes to schedule.dart to avoid crashes if authors field is not specified.
And deleted loremIpsum because it doesn't seem to be used anywhere.

### CODE REVIEW

File: dart_conf_flutter/ios/Podfile.lock [hunk 1]
```diff
@@ -88,40 +88,40 @@ PODS:
     - Flutter
 
 DEPENDENCIES:
-  - cloud_firestore (from `/Users/devoncarew/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.1.0/ios`)
-  - Flutter (from `/Users/devoncarew/projects/flutter/flutter/bin/cache/artifacts/engine/ios`)
-  - url_launcher (from `/Users/devoncarew/.pub-cache/hosted/pub.dartlang.org/url_launcher-1.0.3/ios`)
+  - cloud_firestore (from `/Users/wmleler/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.1.2/ios`)
```
Comment (devoncarew) [inline dart_conf_flutter/ios/Podfile.lock]: We may want to delete this podfile file completely, and then put it in the .gitignore - each dev would then have their own local copy.

File: dart_conf_flutter/lib/src/common.dart [hunk 2]
```diff
@@ -8,9 +8,9 @@ final ThemeData _theme = new ThemeData(primarySwatch: Colors.blue);
 
 ThemeData get appTheme => _theme;
 
-final String loremIpsum =
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. '
-    'Fusce laoreet magna nunc, nec volutpat mi lobortis non.';
+//final String loremIpsum =
```
Comment (devoncarew) [inline dart_conf_flutter/lib/src/common.dart]: I'd just delete instead of commenting out.

File: dart_conf_flutter/lib/src/schedule.dart [hunk 3]
```diff
@@ -131,9 +131,8 @@ class _SchedulePageState extends State<SchedulePage> {
       snapshot['title'],
       snapshot['description'],
       dateTime,
-      duration == null ? null : new Duration(minutes: snapshot['duration']),
-      presenters:
-          (snapshot['authors'] as String).split(',').map((s) => s.trim()),
+      duration == null ? 30 : new Duration(minutes: snapshot['duration']),
```
Comment (devoncarew) [inline dart_conf_flutter/lib/src/schedule.dart]: For this line, what about `new Duration(minutes: snapshot['duration'] ?? 30)`?

File: dart_conf_flutter/lib/src/schedule.dart [hunk 4]
```diff
@@ -158,16 +157,14 @@ class Session implements Comparable<Session> {
...
 
   final String imageUrl;
 
   bool get hasPresenters => presenters?.isNotEmpty ?? false;
 
   String get presentersDescription {
-    return presenters.length > 2
-        ? presenters.join(', ')
-        : presenters.join(' and ');
+    return presenters;
```
Comment (devoncarew) [inline dart_conf_flutter/lib/src/schedule.dart]: you can also remove `presentersDescription` completely and change callers to use `presenters` directly

File: dart_conf_flutter/lib/src/schedule.dart (hunk 3 above)
Comment (wmleler) [reply dart_conf_flutter/lib/src/schedule.dart]: done, and added a static const for the default duration

File: dart_conf_flutter/lib/src/common.dart (hunk 2 above)
Comment (wmleler) [reply dart_conf_flutter/lib/src/common.dart]: done

File: dart_conf_flutter/lib/src/schedule.dart (hunk 4 above)
Comment (wmleler) [reply dart_conf_flutter/lib/src/schedule.dart]: Left presentersDescription, but changed it to return the empty string if no presenters specified.

File: dart_conf_flutter/ios/Podfile.lock (hunk 1 above)
Comment (wmleler) [reply dart_conf_flutter/ios/Podfile.lock]: Added the lock file to .gitignore, but did not delete it (yet).

I redid the iOS stuff, and it worked first time this time. Yippee!

**Comment by wmleler** [comment]
Ok, go over it with a fine toothed comb.

File: dart_conf_flutter/lib/src/schedule.dart [hunk 5]
```diff
@@ -27,6 +27,7 @@ class SchedulePage extends StatefulWidget {
 }
 
 class _SchedulePageState extends State<SchedulePage> {
+  static const int default_duration = 30;
```
Comment (devoncarew) [inline dart_conf_flutter/lib/src/schedule.dart]: nit: `defaultDuration` (but we can address in a follow-up)

**Pull request merged**